### ISSUE TITLE
Display provided address

### ISSUE THREAD

**Issue by jwbos** [IACR/latex#157]
Currently the `\addaffiliation` macro collects information as:

```
\addaffiliation[ror     = 031v4g827,
                onclick = {https://www.nxp.com},
                street  = {Interleuvenlaan 80},
                city    = {Leuven},
                postcode= {3001},
                country = {Belgium}
               ]{NXP Semiconductors}
```

This is used for the meta-information file and *not* used to display the address. 
How to best display the provided information in the pdf?

**Comment by jwbos** [comment]
Something like this?
$affiliation, $street, $postcode, $city, $country
What if authors want to include a department name?
Should we provide a `display` option so the author can format freely?

**Comment by jwbos** [comment]
Alternatively, we can add `name` to `\addaffiliation` and let the main argument (in { }) be the display option in the pdf.

**Comment by kmccurley** [comment]
There are two competing things going on here:
1. authors feel that if they enter something into the LaTeX then it should show up on the page.
2. authors may be unwilling to fill some things out if it shows up on the page. Email addresses are a good example, because I now get spam sent to the iacrcc@digicrime.com email address. Authors have in the past refused to put DOIs in their paper because it ended up taking pages that they would rather use for other things. This is not an issue for CiC, but it's now an ingrained habit of authors.

One problem with #1 is multiple affiliations. The address is for the affiliation that may be shared between authors. and I'd like to avoid repeating the address information for shared affiliations. For an example, [see how the AMS formats papers](https://www.ams.org/journals/cams/2023-03-01/S2692-3688-2023-00016-5/S2692-3688-2023-00016-5.pdf) where they list physical addresses and email addresses at the end of the paper. They end up listing shared affiliations twice. Maybe it doesn't matter since it's compact and included at the end of the paper.

Cambridge university press does something similar; see [this paper](https://www.cambridge.org/core/services/aop-cambridge-core/content/view/4914DB4F8E8305DFC68F9CDCA9D0C8D0/S0960129523000130a.pdf/two-level-type-theory-and-applications.pdf) for example where they list the address with the affiliation. It would be easy to have a formatted address of the affiliation.

We are asking for more information than we need in affiliations, because the only thing I think we really need is country, name, and ROR ID for affiliations. Country is required for listing with Clarivate/Elsevier to show the geographic diversity, and name+ROR is required for [crossref](https://data.crossref.org/schemas/crossref5.3.1.xsd). There are other **optional** elements for affiliations, and that's what we have asked for. These include Acronym for name, city, etc. 

IEEE takes a different approach, putting a mini-biography with a photo of each author at the end of every paper. This seems like overkill and doesn't use any structured information.

**Comment by jwbos** [comment]
I will implement a first attempt. I do sympathize with if an author fills something out then it should show up on the screen. Moreover, I will double check what we require when `version=final` and make sure the documentation is clear.

**Comment by jwbos** [comment]
This is fixed in 9efbfb5.
